src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/jpa/IncidentParty.kt

```diff
@@ -36,27 +31,9 @@ open class IncidentParty(
   @EmbeddedId
   val id: IncidentPartyId,
 
-  // Combination of :
```

Comment:
Role and outcome can now be moved to their appropriate subclasses, as they reference different reference code data (and outcome only applicable to offender party)